java/anotherWayToPrintHelloWorld.java

```diff
@@ -0,0 +1,11 @@
+public class WaysToPrintHello {
```

Comment:
Please add file information as stated in template, like this (on the top of the file)
```sh
// Title  : [Title] 
// Author : [Your-Profile-Username](Your-Profile-Link)
```